### PR TITLE
MattT/APPEALS-9110: Standardize "Ready for review" Modal Buttons and Task Action Labels

### PR DESCRIPTION
**Pull request by ThorntonMatthew**
Resolves [APPEALS-9110](https://vajira.max.gov/browse/APPEALS-9110)

### Description
In order to promote uniformity across all pre-docket workflows, we want all buttons on "Ready for review" modals to have the text "Send". We also would like, for the same reason, for all of the labels for the "Ready for review" task actions to read "Documents ready for X review" where "X" is the organization that receives the appeal upon submitting it as ready for review.

### Acceptance Criteria
- [ ] The submission button for all "Ready for review" modals should have the text "Send".
   - [ ] The modals associated with the following task actions will need their buttons updated:
      - [ ] VHA PO - Ready for Review
      - [ ] VHA CSP - Documents ready for Board Intake review (should already be the case).
      - [ ] EMO - Ready for Review
      - [ ] EDU RPO - Ready for Review
- [ ] The task action labels for all "Ready for review" task actions should be as such:
   - [ ] VHA PO - Documents ready for VHA CAMO review
   - [ ] EMO - Documents ready for Board Intake review
   - [ ] EDU RPO - Documents ready for Board Intake review
   - [ ] VHA CSP - Documents ready for Board Intake review (should already be the case).
- [ ] The titles for all of the modals should read "Ready for review".

### Testing Plan

**Note:** Updating VHA CAMO's modal and task actions have been omitted from this PR as those will be tackled in a separate story. 

#### VHA PO
1. In the Rails console, stage an appeal that will be in a VHA PO's queue:
```ruby
appeal = FactoryBot.create(:appeal, :with_pre_docket_task)
parent_task = appeal.tasks.last
camo_task = VhaDocumentSearchTask.create!( parent: parent_task, appeal: appeal, assigned_at: Time.zone.now, assigned_to: VhaCamo.singleton )
po_task = AssessDocumentationTask.create!( parent: camo_task, appeal: appeal, assigned_at: Time.zone.now, assigned_to: VhaProgramOffice.first )

```
2. Log into Caseflow as a VHA PO user (ex: `VHAPOUSER`)
3. Navigate to the Case Details page for the appeal.
```ruby
appeal.uuid
```
Path: `/queue/appeals/<appeal.uuid>`
4. Ensure the following option is in the actions dropdown: "Documents ready for CAMO team review"
5. Click on "Documents ready for VHA CAMO review". A modal should appear.
6. The modal's title should be "Ready for review"
7. The submission button's text should be "Send"

#### VHA CSP
1. In the Rails console, stage an appeal that will be in the VHA CSP's queue:
```ruby
appeal = FactoryBot.create(:appeal, :with_pre_docket_task)
parent_task = appeal.tasks.last
caregiver_task = VhaDocumentSearchTask.create!( parent: parent_task, appeal: appeal, assigned_at: Time.zone.now, assigned_to: VhaCaregiverSupport.singleton )

```
2. Log into Caseflow as a VHA CSP user (ex: `CAREGIVERUSER`)
3. Navigate to the Case Details page for the appeal.
```ruby
appeal.uuid
```
Path: `/queue/appeals/<appeal.uuid>`
4. Ensure the following option is in the actions dropdown: "Documents ready for Board Intake review"
5. Click on "Documents ready for Board Intake review". A modal should appear.
6. The modal's title should be "Ready for review"
7. The submission button's text should be "Send"

#### EMO
1. In the Rails console, stage an appeal that will be in the EMO's queue:
```ruby
appeal = FactoryBot.create(:appeal, :with_pre_docket_task)
parent_task = appeal.tasks.last
emo_task = EducationDocumentSearchTask.create!( parent: parent_task, appeal: appeal, assigned_at: Time.zone.now, assigned_to: EducationEmo.singleton )

```
2. Log into Caseflow as an EMO user (ex: `EMOUSER`)
3. Navigate to the Case Details page for the appeal.
```ruby
appeal.uuid
```
Path: `/queue/appeals/<appeal.uuid>`
4. Ensure the following option is in the actions dropdown: "Documents ready for Board Intake review"
5. Click on "Documents ready for Board Intake review". A modal should appear.
6. The modal's title should be "Ready for review"
7. The submission button's text should be "Send"

#### EDU RPO
1. In the Rails console, stage an appeal that will be in an EDU RPO's queue:
```ruby
appeal = FactoryBot.create(:appeal, :with_pre_docket_task)
parent_task = appeal.tasks.last
emo_task = EducationDocumentSearchTask.create!( parent: parent_task, appeal: appeal, assigned_at: Time.zone.now, assigned_to: EducationEmo.singleton )
rpo_task = EducationAssessDocumentationTask.create!( parent: emo_task, appeal: appeal, assigned_at: Time.zone.now, assigned_to: EducationRpo.first )
```
2. Log into Caseflow as an EDU RPO user (ex: `EDURPOUSER`)
3. Navigate to the Case Details page for the appeal.
```ruby
appeal.uuid
```
Path: `/queue/appeals/<appeal.uuid>`
4. Ensure the following option is in the actions dropdown: "Documents ready for Board Intake review"
5. Click on "Documents ready for Board Intake review". A modal should appear.
6. The modal's title should be "Ready for review"
7. The submission button's text should be "Send"

- [ ] For higher-risk changes: [Deploy the custom branch to UAT to test](https://github.com/department-of-veterans-affairs/appeals-deployment/wiki/Applications---Deploy-Custom-Branch-to-UAT)

### User Facing Changes
 - [ ] Screenshots of UI changes added to PR & Original Issue

Organization | BEFORE - Dropdown | BEFORE - Modal | AFTER - Dropdown | AFTER - Modal
---|---|---|---|---
VHA PO |||vha-po-dropdown-post-fix-pt2|vha-po-modal-post-fix
VHA CSP |||No changes|csp-modal-post-fix
EMO |||emo-dropdown-post-fix|emo-modal-post-fix
EDU RPO|||edu-rpo-dropdown-post-fix|edu-rpo-modal-post-fix

### CODE REVIEW

**Review by TylerBroyles — CHANGES_REQUESTED** [review]
Overall it looks great. The comments are mostly just general improvements not really specific to the feature.

File: client/app/queue/QueueApp.jsx [hunk 1]
```diff
@@ -1084,33 +1084,33 @@ class QueueApp extends React.PureComponent {
             <PageRoute
               exact
               path={`/queue/appeals/:appealId/tasks/:taskId/${
-                  TASK_ACTIONS.READY_FOR_REVIEW.value
+                  TASK_ACTIONS.VHA_PO_SEND_TO_CAMO_FOR_REVIEW.value
                 }`}
-              title="Ready for Review | Caseflow"
+              title="Ready for review | Caseflow"
```
[the remark this answers is not in the document]
Comment (ThorntonMatthew) [reply client/app/queue/QueueApp.jsx]: I've gone ahead and added them all to PAGE_TITLES just so we have them all in one spot. I feel like it's kind of overkill.. hopefully it pays off.

File: client/app/queue/components/CompleteTaskModal.jsx [hunk 2]
```diff
@@ -358,65 +357,57 @@ const MODAL_TYPE_ATTRS = {
...
+    getContent: ReadyForReviewModal
   },
   vha_caregiver_support_return_to_board_intake: {
     buildSuccessMsg: (appeal) => ({
       title: sprintf(COPY.VHA_CAREGIVER_SUPPORT_RETURN_TO_BOARD_INTAKE_SUCCESS_CONFIRMATION, appeal.veteranFullName)
     }),
     title: () => COPY.VHA_CAREGIVER_SUPPORT_RETURN_TO_BOARD_INTAKE_MODAL_TITLE,
     getContent: VhaCaregiverSupportReturnToBoardIntakeModal,
-    buttonText: COPY.MODAL_RETURN_BUTTON,
     submitButtonClassNames: ['usa-button'],
```
[the remark this answers is not in the document]
Comment (ThorntonMatthew) [reply client/app/queue/components/CompleteTaskModal.jsx]: Ripped that bad boy out of there! Thanks!

File: client/app/queue/components/CompleteTaskModal.jsx [hunk 3]
```diff
@@ -628,11 +618,13 @@ class CompleteTaskModal extends React.Component {
...
   render = () => {
     const modalAttributes = MODAL_TYPE_ATTRS[this.props.modalType];
+    const taskData = taskActionData(this.props);
 
     return (
       <QueueFlowModal
         title={modalAttributes.title(this.getContentArgs())}
-        button={modalAttributes.buttonText}
+        /* eslint-disable-next-line camelcase */
+        button={taskData?.modal_button_text ?? COPY.MODAL_SUBMIT_BUTTON}
```
[the remark this answers is not in the document]
Comment (ThorntonMatthew) [reply client/app/queue/components/CompleteTaskModal.jsx]: Sounds good!

**Pull request merged**